### ISSUE TITLE
Automatic connection no longer working

### ISSUE THREAD

**Issue by f-liva** [steam3d/MagicPods-Windows#189]
Hi

Since I paired my AirPods Max with a second device (but not together with my computer and at different times of the day), MagicPods can no longer automatically connect the headphones. The popup that detects them appears, and if I click the Connect button, they connect, but the connection is no longer automatic.

**Comment by steam3d** [comment]
Hi. This is Windows Bluetooth behavior issue and AirPods Max does not support auto connect. You can check AirPods capability [here](https://help.magicpods.app/fun-capability/).

**Comment by f-liva** [comment]
But if I click on the Connect button shown by MagicPods they connect. So why couldn't MagicPods click this button "for me" and then reconnect the headset without my manual intervention?

**Comment by steam3d** [comment]
But how can i understand that you want to connect your AirPods Max?

**Comment by f-liva** [comment]
When the popup appears the AirPods ara available, so force its connection

**Comment by steam3d** [comment]
You can accidentally touch for headphones and the popup will appear. There is no one event that guarantees that you want to connect headphones.  The best way is add a shortcut to connect

**Comment by f-liva** [comment]
I agree, but this should be decided by the user. I only use the headset with the PC, so I would enable a hypothetical "Force connection as soon as visible" option if it were available.

Should I use the headset with other devices? I disable this option.

I'm just trying to find a solution to restore the auto connection that was happening properly before I connected the headset to a second device.

Is there any way to restore that behavior? Can the headset be reset?

**Comment by steam3d** [comment]
I did not find any way to return this behavior.  After updating the AirPods firmware behavior changed.

**Comment by f-liva** [comment]
Ok no problem
